### PR TITLE
Fix typos

### LINKED ISSUE #310: Fix typos in comments and identifiers across codebase

### PR DESCRIPTION
**Pull request by anton-trunov**
Closes #310

- [x] I have updated CHANGELOG.md
- [x] I have added tests to demonstrate the contribution is correctly implemented: this usually includes both positive and negative tests, showing the happy path(s) and featuring intentionally broken cases
- [x] I have run all the tests locally and no test failure was reported
- [x] I did not do unrelated and/or undiscussed refactorings

### CODE REVIEW

**Review by byakuren-hijiri — DISMISSED** [review]
LGTM!
I'm happy that the names of AST assignments are finally fixed.

**Review by novusnota — DISMISSED** [review]
Neat!

**Comment by novusnota** [comment]
Timeout of math-related tests on Windows :/

**Comment by anton-trunov** [comment]
yeah, had to re-run that

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Built-in function `address()` now handles parse errors correctly: PR [#357](https://github.com/tact-lang/tact/pull/357)
 - All identifiers in error messages are now quoted for consistency: PR [#363](https://github.com/tact-lang/tact/pull/363)
 - The grammar of the unary operators has been fixed, constant and function declarations are prohibited for contracts and at the top level of Tact modules: PR [#365](https://github.com/tact-lang/tact/pull/365)
+- Typos in ABI generation: : PR [#372](https://github.com/tact-lang/tact/pull/372)
```
Comment (Gusarich) [inline CHANGELOG.md:32]: looks like one `:` is extra here

Comment (anton-trunov) [reply CHANGELOG.md:32]: Oops! Well-spotted, will fix in a PR that brushes up the changelog before releasing v1.3.1

**Pull request merged**